modules/serverless-rn-1-31-0.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-31-0_{context}"]
+= Red Hat {ServerlessProductName} 1.31
+
+{ServerlessProductName} 1.31 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-31-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.10.
+* {ServerlessProductName} now uses Knative Eventing 1.10.
+* {ServerlessProductName} now uses Kourier 1.10.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.10.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.10.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+
+* When specifying Kourier for Ingress and using `DomainMapping`, the TLS for OpenShift Route is set to passthrough, and TLS is handled by the Kourier Gateway. Beginning with {ServerlessProductShortName} 1.31, it is possible to specify the enabled cipher suites on the side of the Kourier Gateway.
+
+* Now you can change the pattern for Apache Kafka topic names used by Knative brokers and channels for Apache Kafka.
+
+* The `domain-mapping` deployment in `knative-serving` has been integrated into the `controller` deployment.
+* The `domain-mapping-webhook` deployment in `knative-serving` has been integrated into the `webhook` deployment.
+
+* Integrating {SMProductName} with {ServerlessProductShortName} when Kourier is enabled is now deprecated. Use `net-istio` instead of `net-kourier` for Service Mesh integration.
++
+See the "Integrating {SMProductName} with {ServerlessProductShortName}" section for details.
+
+* The `DomainMapping` `v1alpha1` custom resource definition (CRD) is now deprecated. Use `v1beta1` CRD instead.
+
+[id="fixed-issues-1-31-0_{context}"]
+== Fixed issues
+
+* The `PingSource` adapter metrics, when using OpenShift monitoring, are collected using a different `job` and `service` label value. The previous value was `pingsource-mt-adapter`, the new value is `pingsource-mt-adapter-sm-service`. (FIXME -- What was the issue here? Currently this sounds like it's just a change.)
```

Comment:
It was this one: https://issues.redhat.com/browse/SRVKE-1510
But the current text is misleading, we should add at the beginning that PingSource metrics were unavailable when ServiceMesh integration in full mesh was used with STRICT PeerAuthentication.